src/services/ApiService.ts

```diff
@@ -71,6 +71,25 @@ class ApiService {
 			})
 	}
 
+	/**
+	 * Use the fetch API over HTTP to hit get mastery total endpoint on the server
```

Comment:
Wording was a bit confusing

```suggestion
	 * Use the fetch API over HTTP to hit mastery total endpoint on the server
```